include/clang/AST/Type.h

```diff
@@ -1917,6 +1917,17 @@ class Type : public ExtQualsTypeCommonBase {
   /// \brief Whether this type is or contains a checked type
   bool isOrContainsCheckedType() const;
 
+  enum CheckedValueKind {
+    NoCheckedValue,
+    HasCheckedValue,
+    HasUnCheckedPointer
```

Comment:
Could you change the case of the `C` in HasUnCheckedPointer to lower case (HasUncheckedPointer)?   This matches the capitalization in the rest of the code base.